src/main/java/in/projecteka/gateway/registry/model/CMServiceRequest.java

```diff
@@ -0,0 +1,21 @@
+package in.projecteka.gateway.registry.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+import java.time.LocalDateTime;
+
+@Value
+@Builder
+public class CMServiceRequest {
+    String name;
+    String url;
+    String consentManagerId;
```

Comment:
If consentManagerId is internal (Apparently), why we are expecting this from an admin who just want to update the CM (which can be found by just a suffix)?

Reply:
done